lua/lvim/core/terminal.lua

```diff
@@ -16,7 +16,7 @@ M.config = function()
     insert_mappings = true, -- whether or not the open mapping applies in insert mode
     persist_size = false,
     -- direction = 'vertical' | 'horizontal' | 'window' | 'float',
-    direction = "float",
+    direction = "horizontal",
```

Comment:
I meant this one, i think @ChristianChiarulli preferred it to be float as well

Reply:
Yeah, I figured that it's what you meant. You guys are crazy, I'll put it back to float by default 😄

Reply:
tyvm <3

Reply:
but can you actually hit `<leader>gg` without `lazygit` reacting to the press? e.g. it would try to stage files for example when you press `space`. am I really that slow? :(

Reply:
no i cannot, i usually use `q` to toggle off and `<leader>gg` to toggle on